.vib/minio/cypress/cypress/integration/minio_spec.js

```diff
@@ -8,9 +8,10 @@ it('allows creating a bucket, uploading and retrieving a file', () => {
     bucketName=`${buckets.newBucket.name}.${random}`;
     cy.get('#bucket-name').type(`${bucketName}`);
     cy.contains('button', 'Create Bucket').click();
+    cy.get('a[href="/browser"]').click();
+    cy.get(`span[id="browse-${bucketName}"]`).click();
```

Comment:
Could we get to that page using the `visit()` command? Cypress always recommends doing so as a best practice. I haven't checked this, but maybe something like the following could be a good replacement:

```suggestion
    cy.visit(`/browse-${bucketName}`);
```